lib/adapters/AbstractAdapter.js

```diff
@@ -209,6 +209,11 @@ class AbstractAdapter extends AbstractReaderWriter {
 	}
 
 	_write(resource) {
+		if (!resource.getPath().startsWith(this._virBasePath)) {
+			throw new Error("The path of the resource '" + resource.getPath() + "' does not starts with " +
+				"the configured virtual base path of the adapter '" + this._virBasePath + "'");
```

Comment:
```suggestion
			throw new Error(`The path of the resource '${resource.getPath()}' does not starts with ` +
				`the configured virtual base path of the adapter '${this._virBasePath}'`);
```

Any reason not to use string templates?

Reply:
done